src/app/static/src/app/components/projects/ProjectHistory.vue

```diff
@@ -273,7 +280,10 @@
             currentLanguage: mapStateProp<RootState, Language>(
                 null,
                 (state: RootState) => state.language
-            )
+            ),
+            projects: mapStateProp<ProjectsState, Project[] | null>(namespace, state => {
```

Comment:
I don't really understand why this is necessary - the ProjectsMixin was already providing projects.